librec/src/test/java/librec/util/KernelSmoothingTest.java

```diff
@@ -0,0 +1,107 @@
+package librec.util;
```

Comment:
KernelSmoothing tests here are actually fantastic. A PR with these alone would be decently impressive.